cattenbak.py

```diff
@@ -13,8 +13,11 @@
 discovery_url = "https://discovery.eduroam.app/v1/discovery.json"
 
 
-def get_seq(old_discovery):
-    return int(old_discovery["seq"]) + 1
+def get_serial():
+    old_discovery = requests.get(discovery_url).json()
```

Comment:
Why not use the old_discovery argument, so that get_old_discovery() and get_old_discovery_from_file() both can be used?

Reply:
Updated so that the old discovery is no longer required for calculating the serial.